### PR TITLE
Losses modified

### PR DESCRIPTION
**Pull request by KirillDogadin-std**
Assuming that the losses lacked the calculation that `tf` describes in `define` family of the funcions.

Also the notation with the dictionary was not really clear and had no precise comment. Thus, I've came up with 2 versions that you could've meant. 

Notify me on which one to remove/adjust.

### CODE REVIEW

File: code/pytorch/losses.py [hunk 1]
```diff
@@ -127,16 +127,38 @@ def __init__(self, eye, cv_name_weight, cv_weight, orthogonal_weight):
...
         # same as MultiKENet
-        self.views_cfg = self.views_cfg = {
-            'rv': self,
-            'av': self,
-            'ckgrtv': self,
-            'ckgatv': self,
-            'ckgrrv': self,
-            'ckgarv': self,
-            'cnv': self,
-            'mv': self
```
Comment (KirillDogadin-std) [inline code/pytorch/losses.py:138]: I assume that this was a draft

File: code/pytorch/losses.py [hunk 2]
```diff
@@ -127,16 +127,38 @@ def __init__(self, eye, cv_name_weight, cv_weight, orthogonal_weight):
...
+        self.views_cfg = {
+            'rv': relation_logistic_loss,
+            'av': attribute_triple_loss,
+            'ckgrtv': cross_kg_relation_triple_loss,
+            'ckgatv': cross_kg_attribute_triple_loss,
+            'ckgrrv': cross_kg_relation_reference_loss,
+            'ckgarv': cross_kg_attribute_reference_loss,
+            'cnv': cross_name_view_loss,
+            'mv': mapping_loss
```
Comment (KirillDogadin-std) [inline code/pytorch/losses.py:138]: The asuumed draft change.
As I understand forward just calls a `loss` calculation based on the type while already having previous evaluations done.

File: code/pytorch/losses.py [hunk 3]
```diff
@@ -127,16 +127,38 @@ def __init__(self, eye, cv_name_weight, cv_weight, orthogonal_weight):
...
+        # self.loss = {
+        #     'rv': relation_logistic_loss,
+        #     'av': attribute_triple_loss,
+        #     'ckgrtv': cross_kg_relation_triple_loss,
+        #     'ckgatv': cross_kg_attribute_triple_loss,
+        #     'ckgrrv': cross_kg_relation_reference_loss,
+        #     'ckgarv': cross_kg_attribute_reference_loss,
+        #     'cnv': cross_name_view_loss,
+        #     'mv': mapping_loss
+        # }
```
Comment (KirillDogadin-std) [inline code/pytorch/losses.py:161]: This is how i understood the comment of `# same as MultiKENet`. Doesnt really seem logal. Hit me up if it's actually the correct one and I also need to adjust `forwar`. Or hit me up otherwise so I remove this comment stack.
